poutyne/framework/model.py

```diff
@@ -805,7 +805,16 @@ def predict_on_batch(self, x):
             x = self.preprocess_input(x)
             return torch_to_numpy(self.network(*x))
 
-    def evaluate(self, x, y, *, batch_size=32, return_pred=False, callbacks=None, **dataloader_kwargs):
+    def evaluate(self,
+                 x,
+                 y,
+                 *,
+                 batch_size=32,
+                 return_pred=False,
+                 callbacks=None,
+                 verbose=True,
+                 progress_options: Union[dict, None] = None,
+                 **dataloader_kwargs):
```

Comment:
```suggestion
                 dataloader_kwargs=None):
```
I'm sorry, you're going to have to merge a conflict here since I just pushed this modification into dev.

Reply:
gg not well played